app/build.gradle

```diff
@@ -86,6 +90,7 @@ dependencies {
     implementation 'com.github.salomonbrys.kotson:kotson:2.5.0'
     implementation 'androidx.work:work-runtime-ktx:2.3.4'
     implementation 'androidx.core:core-ktx:1.2.0'
+    implementation 'androidx.activity:activity-ktx:1.2.0-alpha04'
```

Comment:
I had to copy this to my PR because of you suggestion on lifecycleScope and Android Studio told me this is not the latest available version - I'm assuming you're aware and the choice was intentional.

Reply:
Oh, I wasn't aware that this has `lifecycle-runtime-ktx` as a transitive dependency, haha. 
I added this dependency to make use of the new `ActivityResultContracts` in the `LoginActivity`.

Reply:
Now I understood what you meant, I will be updating to a newer version :D

Reply:
Thanks!! :-)